packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -65,6 +65,13 @@ Class name added to the placeholder.
 - Type: `String`
 - Required: No
 
+### icon
+
+Dashicon to display left of the title.
```

Comment:
Maybe we can link to the Dashicon docs https://developer.wordpress.org/resource/dashicons/ and refer that a normal WordPress component could also be passed.

Reply:
Good idea, I've gone ahead and referenced the docs and the normal component as well, specifically mentioning `BlockIcon`, as I've seen it being used in other parts of Gutenberg already for `MediaPlaceholder`.